compute/cloud-client/src/main/java/compute/snapshotschedule/EditSnapshotSchedule.java

```diff
@@ -48,47 +51,49 @@ public static void main(String[] args) throws Exception {
     editSnapshotSchedule(projectId, region, snapshotScheduleName);
   }
 
+  // Edits a snapshot schedule.
   public static Status editSnapshotSchedule(
           String projectId, String region, String snapshotScheduleName)
           throws IOException, InterruptedException, ExecutionException, TimeoutException {
-    String description = "Updated description11";
-    Map<String, String> snapshotLabels = new HashMap<>();
-    snapshotLabels.put("key", "value");
-    ResourcePolicyWeeklyCycleDayOfWeek dayOfWeek = ResourcePolicyWeeklyCycleDayOfWeek.newBuilder()
-            .setDay("Tuesday")
-            .setStartTime("09:00")
-            .build();
-    ResourcePolicyWeeklyCycle weeklySchedule = ResourcePolicyWeeklyCycle.newBuilder()
-            .addDayOfWeeks(dayOfWeek)
-            .build();
-    String onSourceDiskDelete = "apply-retention-policy";
-    int maxRetentionDays = 3;
 
     // Initialize client that will be used to send requests. This client only needs to be created
     // once, and can be reused for multiple requests.
     try (ResourcePoliciesClient resourcePoliciesClient = ResourcePoliciesClient.create()) {
-      ResourcePolicy existingSchedule = resourcePoliciesClient
-              .get(projectId, region, snapshotScheduleName);
+      Map<String, String> snapshotLabels = new HashMap<>();
+      snapshotLabels.put("key", "value");
 
       ResourcePolicySnapshotSchedulePolicySnapshotProperties.Builder snapshotProperties =
-              existingSchedule.getSnapshotSchedulePolicy().getSnapshotProperties().toBuilder();
+              ResourcePolicySnapshotSchedulePolicySnapshotProperties.newBuilder();
       snapshotProperties.putAllLabels(snapshotLabels);
 
+      ResourcePolicyWeeklyCycleDayOfWeek dayOfWeek = ResourcePolicyWeeklyCycleDayOfWeek.newBuilder()
+              .setDay("Tuesday")
+              .setStartTime("09:00")
+              .build();
+      ResourcePolicyWeeklyCycle weeklySchedule = ResourcePolicyWeeklyCycle.newBuilder()
+              .addDayOfWeeks(dayOfWeek)
+              .build();
+
       ResourcePolicySnapshotSchedulePolicySchedule.Builder scheduler =
-              existingSchedule.getSnapshotSchedulePolicy().getSchedule().toBuilder();
+              ResourcePolicySnapshotSchedulePolicySchedule.newBuilder();
       scheduler.clearDailySchedule().clearHourlySchedule();
       scheduler.setWeeklySchedule(weeklySchedule);
 
+      String onSourceDiskDelete = APPLY_RETENTION_POLICY.toString();
```

Comment:
Enums should be prefixed with their class name.

`OnSourceDiskDelete. APPLY_RETENTION_POLICY`

Reply:
Thank you! Fixed